frontend/integration-tests/views/project-status.view.ts

```diff
@@ -0,0 +1,3 @@
+import { $, $$ } from 'protractor';
+
+export const softwareInfo = $('.col-lg-4  .group');
```

Comment:
Here I would suggest to use `$('cluster-overview-cell__info-row')` selector, since its more direct

Reply:
Agree with @jhadvig. This selector is too general and again still assumes a certain page structure. You want to avoid that if you can.

Reply:
I would suggest simply adding a `data-kube-version` attribute to the version string and just use that to test the version. (Testing the label doesn't seem too useful.)